frontend/packages/kubevirt-plugin/src/statuses/vm/vm-status.ts

```diff
@@ -57,6 +62,11 @@ import { PAUSED_VM_MODAL_MESSAGE } from '../../strings/vm/messages';
 const isPaused = (vmi: VMIKind): VMStatusBundle =>
   isVMIPaused(vmi) ? { status: VMStatus.PAUSED, message: PAUSED_VM_MODAL_MESSAGE } : null;
 
+const isDeleting = (vm: VMKind, vmi: VMIKind): VMStatusBundle =>
+  (vm && !!getDeletetionTimestamp(vm)) || (!vm && vmi && !!getDeletetionTimestamp(vmi))
```

Comment:
is there a situation when you have a VMI object without a VM ?

Reply:
yes, it was a user request:
https://issues.redhat.com/browse/KNIP-1165
https://github.com/openshift/console/pull/3841